src/app/shared/models/classes/coffee.class.ts

```diff
@@ -12,6 +12,13 @@ export class Coffee extends Product {
         super(coffeeShape);
 
         if (coffeeShape != null) {
+            this.filterableAttributes = this.filterableAttributes.concat([
```

Comment:
why use concat here? Can you not do just this.filterableAttributes = [...]?

Okay, I see that you are also setting this property in the parent class. You should use the spread operator instead of concat